### PR TITLE
Feature/sof 4713 - spacegroup binary encoding

### PR DESCRIPTION
**Pull request by AcylSilane**

### CODE REVIEW

File: DigitalEcosystem/exploratory/exfoliation/exfoliation_energy.py [hunk 1]
```diff
@@ -0,0 +1,260 @@
...
+
+# In[]:
+
+
+noble_gases = ['He', 'Ne', 'Ar', 'Kr', 'Xe', 'Rn']
+fblock = ['La', 'Ce', 'Pr', 'Nd', 'Pm', 'Sm', 'Eu', 'Gd', 'Tb', 'Dy', 'Ho', 'Er', 'Tm', 'Yb', 'Lu',
+          'Ac', 'Th', 'Pa', 'U',  'Np', 'Pu', 'Am', 'Cm', 'Bk', 'Cf', 'Es', 'Fm', 'Md', 'No', 'Lr']
+d_synths = ['Rf', 'Db', 'Sg', 'Bh', 'HS', 'Mt', 'Ds', 'Rg', 'Cn', 'Nh', 'Fl', 'Mc', 'Lv', 'Ts', 'Og']
+
+bad_elements = noble_gases + fblock + d_synths
```
Comment (timurbazhirov) [inline DigitalEcosystem/exploratory/exfoliation/exfoliation_energy.py]: https://github.com/Exabyte-io/Scientific-Projects/pull/21/files#r698786503

File: DigitalEcosystem/exploratory/exfoliation/exfoliation_energy.py [hunk 2]
```diff
@@ -0,0 +1,260 @@
...
+                'is_hubbard (unitless)',
+                'energy_per_atom (eV)',
+                'bandgap (eV)',
+                'is_bandgap_direct (unitless)',
+                'is_metal (unitless)',
+                'energy_vdw_per_atom (eV/atom)',
+                'total_magnetization (Bohr Magneton)']
+target_column = ['exfoliation_energy_per_atom (eV/atom)']
+matpedia_id = ['2dm_id (unitless)']
+atoms_col = ['atoms_object (unitless)']
```
Comment (timurbazhirov) [inline DigitalEcosystem/exploratory/exfoliation/exfoliation_energy.py]: https://github.com/Exabyte-io/Scientific-Projects/pull/21/files#r698787055

File: DigitalEcosystem/exploratory/exfoliation/exfoliation_energy.py (hunk 1 above)
Comment (AcylSilane) [reply DigitalEcosystem/exploratory/exfoliation/exfoliation_energy.py:58]: moved

File: DigitalEcosystem/exploratory/exfoliation/exfoliation_energy.py (hunk 2 above)
Comment (AcylSilane) [reply DigitalEcosystem/exploratory/exfoliation/exfoliation_energy.py:82]: moved/deleted

File: DigitalEcosystem/exploratory/exfoliation/exfoliation_energy.py (hunk 1 above)
Comment (AcylSilane) [reply DigitalEcosystem/exploratory/exfoliation/exfoliation_energy.py]: Change has been made

File: DigitalEcosystem/exploratory/exfoliation/exfoliation_energy.py (hunk 2 above)
Comment (AcylSilane) [reply DigitalEcosystem/exploratory/exfoliation/exfoliation_energy.py]: Has been done

**Comment by AcylSilane** [comment]
Merging per https://exabyte.atlassian.net/browse/SOF-4713?focusedCommentId=42238

**Pull request merged**